### ISSUE TITLE
WebGPU

### ISSUE THREAD

**Issue by agirault** [Kitware/vtk-js#1966]
http://webgpu.io
https://www.w3.org/TR/webgpu

- [ ] (one-way) feature parity with webgl2 features

**Comment by martinken** [comment]
We have webgpu support to some degree so you probably want to be more specific here. Like a specific feature you want or something. See https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/WebGPU/README.md

**Comment by martinken** [comment]
Ahh you are probably just sticking this in for the raodmap. OK. Gotcha.

**Comment by agirault** [comment]
@martinken indeed! Would be great to use that ticket as a checklist of webGPU functionalities you're planning on bringing to vtk.js if you think that's helpful. You should have permissions to edit my actual comment?

**Comment by daker** [comment]
WebGPU is now enabled in Chrome & Safari
